components/ui/variants.tsx

```diff
@@ -0,0 +1,123 @@
+import { cva } from "class-variance-authority"
+
+export const buttonVariants = cva(
+  "inline-flex items-center justify-center rounded-full font-medium transition-colors focus-visible:outline-none disabled:cursor-not-allowed disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:h-4 [&_svg]:w-4",
+  {
+    variants: {
+      variant: {
+        // ————————————————————— Primary Filled —————————————————————
+        primary_filled: [
+          "bg-keppel-500 text-white shadow-sm",
+          "hover:bg-keppel-400",
+          "focus-visible:ring-2 focus-visible:ring-keppel-500",
+          "active:bg-keppel-700",
+        ].join(" "),
```

Comment:
I think the style guide has background keppel-700 for the default, 500 for the hover and 600 for the active.